src/main/java/com/mraof/minestuck/block/machine/ComputerBlock.java

```diff
@@ -161,16 +236,32 @@ private void dropItems(Level level, int x, int y, int z, BlockState state)
 			float rx = rand.nextFloat() * 0.8F + 0.1F;
 			float ry = rand.nextFloat() * 0.8F + 0.1F;
 			float rz = rand.nextFloat() * 0.8F + 0.1F;
-			ItemEntity entityItem = new ItemEntity(level, x + rx, y + ry, z + rz, ProgramData.getItem(program));
+			ItemStack diskStack = ProgramData.getItem(program);
+			if(!diskStack.isEmpty())
+			{
+				ItemEntity entityItem = new ItemEntity(level, x + rx, y + ry, z + rz, diskStack);
+				entityItem.setDeltaMovement(rand.nextGaussian() * factor, rand.nextGaussian() * factor + 0.2F, rand.nextGaussian() * factor);
+				level.addFreshEntity(entityItem);
+			}
+		}
+		
+		for(int iterate = 0; iterate < be.blankDisksStored; iterate++)
+		{
+			float rx = rand.nextFloat() * 0.8F + 0.1F;
+			float ry = rand.nextFloat() * 0.8F + 0.1F;
+			float rz = rand.nextFloat() * 0.8F + 0.1F;
+			ItemEntity entityItem = new ItemEntity(level, x + rx, y + ry, z + rz, MSItems.BLANK_DISK.get().getDefaultInstance());
 			entityItem.setDeltaMovement(rand.nextGaussian() * factor, rand.nextGaussian() * factor + 0.2F, rand.nextGaussian() * factor);
 			level.addFreshEntity(entityItem);
```

Comment:
After seeing this repeated now in three places, I went and looked for a helper function and found `Containers.dropItemStack()`. I suggest using that one! I would then also recommend using it once with the stack size set to the number of disks to drop, rather than looping over the amount to drop them one at a time.